modules/core/shared/src/main/scala-2/SkunkPlatform.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2018-2021 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package skunk
+
+private[skunk] trait SkunkPlatform {
```

Comment:
Sorry, nit, trying to reserve `Platform` for platform concerns. Hopefully would never be needed...

```suggestion
private[skunk] trait SkunkCompat {
```